cmd/webhook/delete.go

```diff
@@ -0,0 +1,168 @@
+package main
```

Comment:
why have you decided to implement this part of the same binary as sriov-network-operator webhook ?
this is also deleting (cleaning) objects not related to this webhook. 

my preference are either:
1. new binary
2. new mode for the controller manager binary